ts/components/registration/stages/RestoreAccount.tsx

```diff
@@ -117,6 +117,8 @@ async function signInWithNewDisplayName({
   }
 }
 
+let abortController = new AbortController();
```

Comment:
const?

Reply:
The abortController gets reassigned when we try to poll otherwise there is no way to update the signal from aborted if we fail once. Trust me bro ™️